doc/lua-filters.md

```diff
@@ -305,6 +305,24 @@ end
 This makes it possible to apply these functions on strings using
 colon syntax (`mystring:uc_upper()`).
 
+# Debugging Lua filters
+
+It is possible to use a debugging interface to halt execution and step 
+through a Lua filter line by line as it is run inside Pandoc. This is 
+accomplished using the remote-debugging interface of the package 
+[`mobdebug`](https://github.com/pkulchenko/MobDebug). Although mobdebug 
+can be run from the terminal, it is more useful run within the 
+donation-ware Lua editor and IDE, [Zerobrane](https://studio.zerobrane.com/). 
+Zerobrane offers a REPL console and UI to step-through and view all 
+variables and state.
+
+If you already have Lua 5.3 installed, you can add `mobdebug` and its 
+dependency `luasocket` using [`luarocks`](https://luarocks.org), which 
+should then be available on the path. Zerobrane also includes both of these 
+in its package, so if you don't want to install Lua seperately, you should 
+modify your lua path and cpath to include the correct locations; 
```

Comment:
Small typo: Lua should be capitalized; or maybe mention the environment variables directly?

Reply:
Hi @tarleb, I've added the correct env names, and also linked to the dependent packages on luarocks. I am hard-wrapping the paragraphs at 80 chars, is that OK?

Reply:
Great, thanks!

80 chars is ok. Most of the text uses a limit of 66, but anything below 80 works fine.